src/Apps/Order/Routes/Payment/OrderPaymentContext.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useReducer, useContext, createContext, Dispatch } from "react"
+import { CommercePaymentMethodEnum } from "__generated__/Payment_order.graphql"
+import { orderPaymentReducer } from "./orderPaymentReducer"
+import createLogger from "Utils/logger"
+
+const logger = createLogger("[dev: OrderPaymentContext] state:")
+
+/* ACTIONS ------------------------------------------------------------------ */
```

Comment:
This is super minor and feel free to ignore, but we don't usually see this kind of comment form throughout the codebase. Something as simple as 

```
/**
 * Actions 
 */ 
```

Is sufficient